client/components/RelatedAndOutfit/Outfit.jsx

```diff
@@ -1 +1,21 @@
-import React from 'react';
\ No newline at end of file
+import React from 'react';
+import exampleProductDetails12012 from './exampleData.js';
+
+class RelatedItems extends React.Component {
+  constructor(props) {
```

Comment:
Is this supposed to be named 'Outfit'?

Reply:
Haha yes - this widget is two carousel widgets, one is for items related to whatever is displayed at the top, and another is where the user can select products to add to their "outfit".